.eslintrc.js

```diff
@@ -32,6 +32,7 @@ module.exports = {
         'space-in-parens'                   : ['error', 'never'],
         'space-infix-ops'                   : 'error',
         'space-unary-ops'                   : 'error',
+        'no-multiple-empty-lines'           : ["error", { "max": 1, "maxEOF": 1 }],
```

Comment:
Not an issue, but would appreciate it if it follows the same style currently used here. 🙂 
(to put in alphabetical order and use single quotes)